transport/transport-runtime/src/androidTest/java/com/google/android/datatransport/runtime/UploaderIntegrationTest.java

```diff
@@ -277,7 +277,7 @@ public void uploader_withPendingClientMetricsAndSuccessfulUpload_shouldResetCoun
                 new EncodedPayload(
```

Comment:
I think we also want a integration test with the changes.

Reply:
Done